ogr/ogrsf_frmts/miramon/mm_gdal_functions.c

```diff
@@ -100,30 +100,21 @@ void MM_FillFieldDescriptorByLanguage(void)
 {
     MM_strnzcpy(szInternalGraphicIdentifierEng, "Internal Graphic identifier",
                 MM_MAX_IDENTIFIER_SIZE);
-    MM_strnzcpy(szInternalGraphicIdentifierCat, "Identificador Grafic intern",
+    MM_strnzcpy(szInternalGraphicIdentifierCat, "Identificador Gràfic intern",
```

Comment:
@AbelPau I strongly suspect this commit is wrong, as you now use UTF-8 characters for the accentuated characters, whereas apparently Latin1 encoding was expected. So you would likely need a MM_strnzcpy_UTF8_to_Latin1() helper function. Or just revert, and use things like
```
*(unsigned char*)&szInternalGraphicIdentifierCat[16] = MM_a_WITH_GRAVE;
```
to make the code work with unsigned MM_ constants

Reply:
Ok, let's try

Reply:
It seems it worked